### PR TITLE
docs: add software dist FAQ

### PR DESCRIPTION
**Pull request by jamesacklin**
Draft PR for the software dist FAQ.

Original gist from @eamsden:
https://gist.github.com/eamsden/359a2ed005502dcca8edf9213533772a

Base hashes forthcoming and marked as "TK."

### CODE REVIEW

**Comment by eamsden** [comment]
When we have the OTA ready we should update this with base hashes in the table

File: content/using/os/dist-faq.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+If the developer provides a version of the app over software distribution and you install it, the new app will migrate the old state.
+
+**What's the base hash?**
+
+After this OTA there is a base hash _per app desk_:
+
+| Software     | Desk       | Base Hash |
+| ------------ | ---------- | --------- |
+| Arvo/OS      | %base      | TK        |
```
Comment (eamsden) [inline content/using/os/dist-faq.md]: This needs base hashes before publication

**Review by eamsden — CHANGES_REQUESTED** [review]
using request changes to put a hold on this until we have base hashes

**Comment by eamsden** [comment]
Also the /os/dist-faq path returns 404 on the preview deployment

**Comment by jamesacklin** [comment]
Works on my machine(tm): https://urbit-org-k53c3g9ki-urbit.vercel.app/using/os/dist-faq

**Comment by eamsden** [comment]
> Works on my machine(tm): https://urbit-org-k53c3g9ki-urbit.vercel.app/using/os/dist-faq

yeah sorry was a pebkac

**Review by eamsden — APPROVED** [review]
hashes are now in place

**Pull request merged**